### PR TITLE
Update icon library

### PR DESCRIPTION
**Pull request by grahambancroft**
## Done

- Update icon library with latest VF icons
- Add titles to image to make identification easier
- Add static images to assets folder

## QA

- Pull branch, run npm install and gulp build
- Go to the demo - demo/index.html#icons to see the changes
- Make sure the icons listed in the card have been updated by comparing them to the icons at [http://ubuntudesign.github.io/maas-gui-vanilla-theme/#icons](http://ubuntudesign.github.io/maas-gui-vanilla-theme/#icons)

## Details

Fixes https://app.zenhub.com/workspace/o/canonicalltd/maas-design/issues/183

### CODE REVIEW

**Comment by kwm14** [comment]
@grahambancroft could you attach a screenshot of your updates as I'm not able to view the demo. Thanks

File: demo/index.html [hunk 1]
```diff
@@ -465,12 +468,12 @@ <h5 class="u-align--right">Example</h5>
...
-                                    <i class="icon u-margin--right-small icon--success"></i>
-                                    <i class="icon u-margin--right-small icon--success icon--small"></i>
+                                    <i title="u-margin--right-small icon--success icon--huge" class="icon u-margin--right-small icon--success icon--huge"></i>
+                                    <i title="u-margin--right-small icon--success icon--gigantic" class="icon u-margin--right-small icon--success icon--gigantic"></i>
+                                    <i title="u-margin--right-small icon--success icon--mega" class="icon u-margin--right-small icon--success icon--mega"></i>
+                                    <i title="u-margin--right-small icon--success icon--large" class="icon u-margin--right-small icon--success icon--large"></i>
+                                    <i title="u-margin--right-small icon--success" class="icon u-margin--right-small icon--success"></i>
+                                    <i title="u-margin--right-small icon--success icon--small" class="icon u-margin--right-small icon--success icon--small"></i>
```
Comment (anthonydillon) [inline demo/index.html]: Updated these titles

File: demo/index.html [hunk 2]
```diff
@@ -1028,14 +1031,14 @@ <h5 class="u-align--right">Example</h5>
                                             <label class="form__group-label" for="exampleInputEmail407">Email address</label>
                                             <div class="form__group-input">
                                                 <input type="email" id="exampleInputEmail407" placeholder="Email">
-                                                <i class="icon icon--success-grey"></i>
+                                                <i title="success-grey" class="icon icon--success-grey"></i>
```
Comment (anthonydillon) [inline demo/index.html]: success-grey is probably not a great title for this icon.

File: demo/index.html [hunk 3]
```diff
@@ -1717,7 +1720,7 @@ <h5 class="u-align--right">Example</h5>
                                                 <div class="table__dropdown">
                                                     <div class="table__dropdown-title">
                                                         <h2 class="u-float--left">Editing</h2>
-                                                        <i class="icon icon--remove u-float--right u-margin--top-small u-margin--right-small"></i>
+                                                        <i title="remove u-float--right u-margin--top-small u-margin--right-small" class="icon icon--remove u-float--right u-margin--top-small u-margin--right-small"></i>
```
Comment (anthonydillon) [inline demo/index.html]: Please updated this title

File: demo/index.html [hunk 4]
```diff
@@ -2664,7 +2667,7 @@ <h5 class="u-align--right">Example</h5>
                                     <div class="page-header u-padding--left u-padding--right">
                                         <div class="wrapper--inner">
                                             <h1 class="page-header__title">MAAS test title</h1>
-                                            <p class="page-header__status">Ready <span class="u-margin--left-small u-text--on"><i class="icon u-margin--right-tiny icon--power-on"></i>Power on</span><a href="" class="page-header__status-check">check now</a></p>
+                                            <p class="page-header__status">Ready <span class="u-margin--left-small u-text--on"><i title="u-margin--right-tiny icon--power-on" class="icon u-margin--right-tiny icon--power-on"></i>Power on</span><a href="" class="page-header__status-check">check now</a></p>
```
Comment (anthonydillon) [inline demo/index.html]: This one too

File: demo/index.html [hunk 5]
```diff
@@ -2686,7 +2689,7 @@ <h1 class="page-header__title">MAAS test title</h1>
                                     <div class="page-header u-padding--left u-padding--right">
                                         <div class="wrapper--inner">
                                             <h1 class="page-header__title">MAAS test title</h1>
-                                            <p class="page-header__status">Ready <span class="u-margin--left-small u-text--error"><i class="icon u-margin--right-tiny icon--power-error"></i>Power error</span><a href="" class="page-header__status-check">check now</a></p>
+                                            <p class="page-header__status">Ready <span class="u-margin--left-small u-text--error"><i title="u-margin--right-tiny icon--power-error" class="icon u-margin--right-tiny icon--power-error"></i>Power error</span><a href="" class="page-header__status-check">check now</a></p>
```
Comment (anthonydillon) [inline demo/index.html]: This one too

File: demo/index.html [hunk 6]
```diff
@@ -2708,7 +2711,7 @@ <h1 class="page-header__title">MAAS test title</h1>
                                     <div class="page-header u-padding--left u-padding--right">
                                         <div class="wrapper--inner">
                                             <h1 class="page-header__title">MAAS test title</h1>
-                                            <p class="page-header__status">Ready <span class="u-margin--left-small u-text--off"><i class="icon u-margin--right-tiny icon--power-off"></i>Power off</span><a href="" class="page-header__status-check">check now</a></p>
+                                            <p class="page-header__status">Ready <span class="u-margin--left-small u-text--off"><i title="u-margin--right-tiny icon--power-off" class="icon u-margin--right-tiny icon--power-off"></i>Power off</span><a href="" class="page-header__status-check">check now</a></p>
```
Comment (anthonydillon) [inline demo/index.html]: This one too

File: demo/index.html [hunk 7]
```diff
@@ -2730,7 +2733,7 @@ <h1 class="page-header__title">MAAS test title</h1>
                                     <div class="page-header u-padding--left u-padding--right u-margin--none">
                                         <div class="wrapper--inner">
                                             <h1 class="page-header__title">MAAS test title</h1>
-                                            <p class="page-header__status">Ready <span class="u-margin--left-small u-text--loading"><i class="icon u-margin--right-tiny icon--loading u-animation--spin"></i>Loading</span><a href="" class="page-header__status-check">check now</a></p>
+                                            <p class="page-header__status">Ready <span class="u-margin--left-small u-text--loading"><i title="u-margin--right-tiny icon--loading u-animation--spin" class="icon u-margin--right-tiny icon--loading u-animation--spin"></i>Loading</span><a href="" class="page-header__status-check">check now</a></p>
```
Comment (anthonydillon) [inline demo/index.html]: This one too

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Some titles need fixing.

**Comment by grahambancroft** [comment]
@kwm14 Screenshot as requested

screen shot 2017-09-26 at 16 52 10

**Comment by kwm14** [comment]
## Visual review

- remove error outline icon - `icon icon--error`
- remove duplicate tooltip icon, first column third row down - `icon icon--tooltip`
- remove tick outline icon - `icon icon--pass`

icon-board

**Comment by grahambancroft** [comment]
@kwm14 Done

screen shot 2017-09-26 at 17 52 54

**Comment by kwm14** [comment]
@grahambancroft 

You've removed the wrong 'error' icon from the icon set.
- [ ] remove error outline icon
screen shot 2017-09-26 at 13 28 27

- [ ] add error fill icon
screen shot 2017-09-26 at 13 28 38

**Comment by grahambancroft** [comment]
@kwm14 I removed icon--error which is the one you asked me to remove, I missed the outline bit, power--error is the outline icon, I'll remove that one and get icon--error back in there :)

**Comment by kwm14** [comment]
@grahambancroft ok cool. Ah I see that must of had the same class name :)

**Comment by grahambancroft** [comment]
@kwm14 Done
screen shot 2017-09-26 at 21 04 58

**Comment by kwm14** [comment]
@grahambancroft looking good 👍

**Review by anthonydillon — APPROVED** [review]
:+1: Thanks

**Pull request merged**